docs/Using-Pantheon/Transactions/Transaction-Pool.md

```diff
@@ -14,11 +14,20 @@ the maximum number of hours to retain pending transactions in the transaction po
 * [`newPendingTransactions`](../../Pantheon-API/RPC-PubSub.md#pending-transactions) and [`droppedPendingTransactions`](../../Pantheon-API/RPC-PubSub.md#dropped-transactions)
 RPC subscriptions to notify of transactions added to and dropped from the node transaction pool  
 
+## Dropping Transactions when Transaction Pool Full 
+
 Once full, the Pantheon transaction pool accepts and retains local transactions in preference to remote transactions. 
 If the transaction pool is full of local transactions, the oldest local transactions are dropped first.  That is, a 
 full transaction pool continues to accept new local transactions by first dropping remote transactions and then by 
 dropping the oldest local transactions. 
 
+## Replacing Transactions with Same Nonce
+
+Submitting a transaction with the same sender and nonce as a pending transaction and a higher gas price replaces the pending transaction
+with the new one.
```

Comment:
This applies equally to local and remote transactions but "submitting" makes it sound like a local transaction issue only.